src/spoof.ts

```diff
@@ -87,15 +98,16 @@ const getElementBox = async (page: Page, element: ElementHandle, relativeToMainF
   }
 }
 
-export function path (point: Vector, target: Vector, spreadOverride?: number)
-export function path (point: Vector, target: BoundingBox, spreadOverride?: number)
-export function path (start: Vector, end: BoundingBox | Vector, spreadOverride?: number): Vector[] {
+export function path (point: Vector, target: Vector, spreadOverride?: number, moveSpeed?: number | null)
+export function path (point: Vector, target: BoundingBox, spreadOverride?: number, moveSpeed?: number | null)
+export function path (start: Vector, end: BoundingBox | Vector, spreadOverride?: number, moveSpeed?: number | null): Vector[] {
   const defaultWidth = 100
   const minSteps = 25
   const width = 'width' in end ? end.width : defaultWidth
   const curve = bezierCurve(start, end, spreadOverride)
   const length = curve.length() * 0.8
-  const baseTime = Math.random() * minSteps
+  const speed = moveSpeed == null ? Math.random() : (25 / moveSpeed)
```

Comment:
this 25 here should probably refer to `minSteps`, right?

Reply:
Actually not. `25` was chosen to get `½` when `moveSpeed = 50`

Reply:
Why did I choose `50` to be the average?

First I used `1/moveSpeed`, but it didn't feel nice because `moveSpeed = 2` was the default's average.
Since you are a maintainer, you can of course decide which values to use.

Reply:
Alright, seems fine then :+1: